Makefile

```diff
@@ -1,6 +1,25 @@
 GOBASE=$(shell pwd)
 GOBIN=$(GOBASE)/bin
 GO_BUILD_FLAGS :=-tags 'containers_image_openpgp osusergo exclude_graphdriver_btrfs exclude_graphdriver_devicemapper'
+ROOT_DIR := $(or ${ROOT_DIR},$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST)))))
+
+VERBOSE ?= false
+REPORTS ?= $(ROOT_DIR)/reports
+
+GO_TEST_FORMAT = pkgname
+ifeq ($(VERBOSE), true)
+        GO_TEST_FORMAT=standard-verbose
+endif
+
+GINKGO_REPORTFILE := $(or $(GINKGO_REPORTFILE), ./junit_unit_test.xml)
+GO_UNITTEST_FLAGS = --format=$(GO_TEST_FORMAT) $(GOTEST_PUBLISH_FLAGS) -- -count=1 -cover
```

Comment:
Please add the  $(GO_BUILD_FLAGS), so we do not depend on gpgme-devel and the BTRFS and Device Mapper graph drivers.

Reply:
Thanks.